modules/api/php/models/candidatesrow.class.inc

```diff
@@ -24,7 +24,6 @@ class CandidatesRow implements \LORIS\Data\DataInstance
 {
     private $_candid;
     private $_projectname;
-    private $_projectid;
```

Comment:
Why is this removed? It is used in https://github.com/aces/Loris/pull/6775/files#diff-e8883d6dbf607c4d208c6405a42a0f5b9d044d9e2e705ec5cfaca41703943627R91